code_search/src/code_search/dataflow/cli/arguments.py

```diff
@@ -29,7 +29,7 @@ def add_parser_arguments(parser):
                       help='BigQuery dataset for output results')
   additional_args_parser.add_argument('--pre_transformed', action='store_true',
                       help='Use a pre-transformed BigQuery dataset')
-  additional_args_parser.add_argument('--wait_until_finish', action='store_true',
+  additional_args_parser.add_argument('--wait_until_finished', action='store_true',
```

Comment:
Comment from #323
Does Dataflow's built-in flag "wait_until_finish" flag do what we want? Can we get rid of our custom flag?

Reply:
Do you know the answer?